Reviewer/src/main/java/com/softserveinc/reviewer/annotation/TestDatabase.java

```diff
@@ -11,4 +11,4 @@
 @Retention(RUNTIME)
 @Target({ ElementType.FIELD, ElementType.PARAMETER })
 @BindingAnnotation
-public @interface Collection {}
+public @interface TestDatabase {}
```

Comment:
Let's try DatabaseName instead. This configuration name should be valid for production db as well. And TestDatabase would be a strange naming for it.